### DISCUSSION TITLE
ScriptReference/Resources.Load

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#461] [Notes]
# ScriptReference/Resources.Load

https://docs.unity3d.com/ScriptReference/Resources.Load.html

**Comment by SoylentGraham** [comment]
I spent a while trying to make this work with assets in a package. (prefixing with `Packages/com.package.xyz/` etc)
Asset located at 
`<yourpackage>/Runtime/Resources/Bees.mat` 
Loads with 
`Resources.Load("Bees")`
